candidates/views.py

```diff
@@ -998,6 +998,8 @@ def get_context_data(self, **kwargs):
         total_unfinished = 0
         for progress_item in progress.values():
             for req_progress in progress_item['requirements']:
+                if not req_progress['requirement']:
+                    continue
```

Comment:
Are there any implications of just "continuing" for an empty Requirement in req_progress?
Just curious and making sure you thought about this

Reply:
I saw a similar structure elsewhere in the code. It looks like the req_progress variables with an empty requirement don't have much purpose / might have been added accidentally? They probably don't need to exist

Reply:
Ok, yeah now that you mentioned it
When I investigated the issue, the specific requirement was not filled in which resulted in the None that was included in the dictionary

Agreed and OK to skip it then